test/component/summary/testExecutorsSection.js

```diff
@@ -99,30 +98,31 @@ describe('summary-executor-section', () => {
         });
 
         it('test data is played back correctly on the summary page executors section', (done) => {
+            const executorsData = require('test/data/summary-executors');
             testWrapper.agent.post('/prepare-session/form')
                 .send(sessionData)
                 .end((err) => {
                     if (err) {
                         throw err;
                     }
 
+                    delete require.cache[require.resolve('test/data/summary-executors')];
                     const playbackData = {
                         firstName: applicantContent.name.firstName,
                         lastName: applicantContent.name.lastName,
                         phoneNumber: applicantContent.phone.phoneNumber,
-                        applicantAddress: applicantContent.address.question
+                        applicantAddress: applicantContent.address.question,
+                        address: executorsData.applicant.address.formattedAddress,
+                        exec2fullName: executorsData.executors.list[1].fullName,
+                        exec2IsApplying: executorsData.executors.list[1].isApplying ? 'Yes' : 'No',
+                        exec2HasAlias: executorsData.executors.list[1].hasOtherName ? 'Yes': 'No',
+                        exec2Alias: executorsData.executors.list[1].currentName,
+                        exec2AliasReason: executorsData.executors.list[1].currentNameReason,
+                        exec3fullName: executorsData.executors.list[2].fullName,
+                        exec3IsApplying: executorsData.executors.list[2].isApplying ? 'Yes' : 'No',
+                        exec3NotApplyingReason: executorsData.executors.list[2].notApplyingReason
```

Comment:
need to add additional data to this test for a deceased executor ie "isDead": true,